content/about/mirrors.md

```diff
@@ -1,50 +1,50 @@
 ---
 title: "GRASS GIS Website mirrors"
-date: 2022-05-28T10:12:05+02:00
+date: 2025-05-11T10:12:05+02:00
 layout: "general"
 ---
 
-### Official main site via HTTP/FTP
+### Official main site
 
 * [Official main site: grass.osgeo.org](https://grass.osgeo.org/) (Portland, USA)
 
 ### African Web site mirrors
 
-* [South Africa](https://grass.mirror.ac.za/) (Tier 1,TENET)
-* [Morocco](https://grass.marwan.ma/) (Moroccan National Research and Education Network, MARWAN)
+* [Morocco](https://grass.marwan.ma/) (Tier 1, Moroccan National Research and Education Network, MARWAN)
+* [South Africa](https://grass.mirror.ac.za/) (Tier 1, TENET)
```

Comment:
What does Tier 1 mean?

Reply:
- Tier 1: direct sync from main server grass.osgeo.org
- Tier 2: sync from a mirror server

I can remove this 'Tier X' thing, which is a relic from the past.